assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -451,9 +452,18 @@ public void handleAccess(HttpServletRequest req, HttpServletResponse res, Refere
                     // fail the request, user not logged in yet.
                 } else {
                     if (REF_TYPE_SUBMISSIONS.equals(ref.getSubType())) {
+                        String assignmentName = "bulk_download";
+                        try {
+                            assignmentName = getAssignment(ref.getReference()).getTitle();
+                        } catch (Exception ignore) {
+                            log.error("Could not find assignment for ref = {}", ref.getReference());
+                        }
+
+                        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd_HH-mm");
```

Comment:
use the following method as its internationalized: 
```
getUsersLocalDateTimeString(Instant.now())
```

Reply:
Internationalized is bad in this case, because "/" is not fun in filenames.